tt1-blocks/experimental-theme.json

```diff
@@ -188,6 +188,9 @@
 					"unit": "20px",
 					"horizontal": "25px",
 					"vertical": "30px"
+				},
+				"font-weight":{
+					"title": "300"
```

Comment:
I prefer this approach, but I'd recommend we call this `light` instead of `title`, just in case we use it outside of just titles.

Reply:
meh where did I get "thin" from -switching again, to "light".